demos/TicTacToe/Classes/TicTacToeLayer.h

```diff
@@ -22,57 +42,65 @@ using firebase::Future;
 using firebase::database::DataSnapshot;
 using firebase::database::MutableData;
 using firebase::database::TransactionResult;
+using std::string;
 
 static const int kTilesX = 3;
 static const int kTilesY = 3;
 class SampleValueListener;
 class ExpectValueListener;
```

Comment:
Add a new line after ExpecatValueListener and then add a comment block before class TickTacToeLayer which describes what it is.

Everything that you declare in a header file should have some comment next to it.  I would say that the MainMenuScene.h header has good comment coverage.